doc/classes/Navigation2DServer.xml

```diff
@@ -188,6 +188,28 @@
 				Returns the navigation path to reach the destination from the origin, while avoiding static obstacles.
 			</description>
 		</method>
+		<method name="map_get_closest_point" qualifiers="const">
+			<return type="Vector2">
+			</return>
+			<argument index="0" name="map" type="RID">
+			</argument>
+			<argument index="1" name="point" type="Vector2">
+			</argument>
+			<description>
+				Returns the nearest point on the navigation mesh surface nearest to the submitted point.
```

Comment:
Same comment as for the `Navigation2D` equivalent.

Reply:
done